src/Raven.Server/Documents/AI/ChatCompletionClient.cs

```diff
@@ -981,7 +981,7 @@ public static class RequestFields
             public const string OpenAiProject = "OpenAI-Project";
             public const string MediaTypeApplicationJson = "application/json";
 
-            public const string DefaultRelativeUri = "/v1/chat/completions";
+            public const string DefaultRelativeUri = "v1/chat/completions";
```

Comment:
Please check that we properly behave here if the `url` ends with `/` or not.
Pretty sure it _would_ behave properly, but important to check